lib/css/components/_stacks-code-blocks.less

```diff
@@ -99,3 +99,18 @@ code[class*="language-"] {
         font-weight: bold;
     }
 }
+
+pre.s-code-block .s-code-block--line-numbers {
+    float: left;
```

Comment:
Did we want to avoid floating here? Probably more appropriate to punt this to later.

Reply:
Yeah, most definitely, but I need to do a little exploration on how to get this to work on both our docs (where the block itself is not accessible at render time to check for `linenums` class name) and in prod where we highlight a block directly (and can check its classes).